src/gui/properties/peerlistwidget.cpp

```diff
@@ -140,7 +139,6 @@ PeerListWidget::PeerListWidget(PropertiesWidget *parent)
 PeerListWidget::~PeerListWidget()
 {
     saveSettings();
-    delete m_proxyModel;
```

Comment:
>but conceptually this is wrong

You mean in the sense of destructor order, then okay.
Well I think I should just revert, too much hassle down the way.